src/app/my-button/my-button.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+import { AddinClientInitArgs } from '@blackbaud/sky-addin-client';
+import { AddinClient } from '@blackbaud/sky-addin-client';
+
+@Component({
+  selector: 'my-button',
+  templateUrl: './my-button.component.html'
+})
+export class MyButtonComponent implements OnInit {
+
+  private addinClient: AddinClient;
+
+  public ngOnInit() {
+
+    this.addinClient = new AddinClient({
+      callbacks: {
+        init: (args: AddinClientInitArgs) => {
+          args.ready({
+            showUI: true,
+            title: 'Scan check',
```

Comment:
Let's change the title here to reflect what will actually happen when the button is clicked - suggestion "Add customer"